--- sites/default/default.settings.php
+++ sites/default/default.settings.php
@@ -444,3 +444,10 @@ ini_set('session.cookie_lifetime', 2000000);
  * Remove the leading hash signs to disable.
  */
 # $conf['allow_authorize_operations'] = FALSE;
+
+/**
+ * Allow for local overrides by looking for a file named settings.local.php.
+ */
+if (file_exists(dirname(__FILE__) . '/settings.local.php')) {
+  require_once(dirname(__FILE__) .'/settings.local.php');
+}
